static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -197,7 +197,7 @@ $fc-item-gutter: $gs-gutter / 4;
     margin-right: $gs-gutter / 4;
     font-weight: 700;
     border-right-style: solid;
-    border-right-width: 1px;
+    border-width: 0 1px 0 0;
```

Comment:
where was the 3px coming from?

Reply:
It looks like a Safari-only issue: https://jsfiddle.net/8t0n2u6s/

Reply:
also seeing it in FF 😂 that's weird.

Reply:
ooooh, turns out [initial value of `border-width` is `medium`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-width)! who knew?

Reply:
maybe just `border-right: 1px solid transparent` then?